testing/sdk_diag.py

```diff
@@ -82,7 +82,11 @@ def handle_test_setup(item: pytest.Item):
         _testlogs_current_test_suite = test_suite
         global _testlogs_ignored_task_ids
         _testlogs_ignored_task_ids = _testlogs_ignored_task_ids.union(
-            [task.id for task in sdk_tasks.get_summary(with_completed=True)]
+            [
+                task.id
+                for task in sdk_tasks.get_summary(with_completed=True)
+                if not _task_whitelist_callback(task)
```

Comment:
The function `_task_whitelist_callback()` returns a callback and it's not sure how this is evaluated here.

Reply:
this is working as expected, what do you mean by that ?